libbeat/docs/shared-tls-logstash-config.asciidoc

```diff
@@ -22,7 +22,8 @@ To use TLS mutual authentication:
 {beatname_uc} and Logstash. Creating a correct SSL/TLS infrastructure is outside the scope of this
 document. There are many online resources available that describe how to create certificates.
 +
-NOTE: Certificates must be signed by your root CA. Intermediate CAs are currently not supported. 
+NOTE: Certificates must be signed by your root CA. Intermediate CAs are currently not supported. You may be able to
+use intermediate CAs by merging all the certificates in the chain into one file.
```

Comment:
Given that this statement is being made in the context of signing client certificates, I don't think that last sentence is relevant. When you sign a client certificate, you sign it with one, and only one, CA certificate.

That statement is relevant when talking about trusting a chain of certificate authorities. So it would be more relevant under the beat's `certificate_authorities` configuration option. With the logstash beats input plugin being re-written in Java with Netty, hopefully the server-side problem that required this workaround of concatenating root CA and intermediate CA into one file is no longer required on the Beats side. Probably @ph has details on this.

Related: https://github.com/elastic/beats/issues/1494